rgw/v2/tests/s3_swift/test_rgw_mfa.py

```diff
@@ -0,0 +1,251 @@
+import os, sys
+
+sys.path.append(os.path.abspath(os.path.join(__file__, "../../../..")))
+from v2.lib.resource_op import Config
+import v2.lib.resource_op as s3lib
+from v2.lib.s3.auth import Auth
+from v2.utils.log import configure_logging
+import v2.utils.utils as utils
+from v2.utils.utils import RGWService
+from v2.tests.s3_swift import reusable
+from v2.lib.rgw_config_opts import CephConfOp, ConfigOpts
+from v2.utils.utils import HttpResponseParser
+import traceback
+import argparse
+from v2.lib.exceptions import TestExecError, RGWBaseException
+from v2.utils.test_desc import AddTestInfo
+from v2.lib.s3.write_io_info import IOInfoInitialize, BasicIOInfoStructure , BucketIoInfo
+import json, time, hashlib
+import logging
+import random
+import v2.lib.manage_data as manage_data
+
+log = logging.getLogger()
+TEST_DATA_PATH = None
+password = "32characterslongpassphraseneeded".encode('utf-8')
+encryption_key = hashlib.md5(password).hexdigest()
```

Comment:
do you need password and encryption here. I dont see they are being used here.

Reply:
agree, removed the password and encryption_key